### PR TITLE
Feat/add back ts mft

### LINKED ISSUE #287: Need for test for time series (temperature/mass flow rate) model
add (old?) model back and then add test

It seems as though there is no longer a test for the model that uses data in the form of a time series of temperatures and mass flow rates. The current time series test addresses the model that uses this data in this file (time_series_large_office_2013_5B.mos). It would be good to have a test for the time series model using the data in this file (ts_mft_ets_example_data.mos). I recently updated that data, documented in this PR: 
https://github.com/urbanopt/geojson-modelica-translator/pull/285

Thanks, Nate, for identifying this. @vtnate 

Thanks! @nllong @macintoshpie (Tagging you as an FYI, Nick and Ted.)

### PR DESCRIPTION
**Pull request by macintoshpie**
#### Any background context you want to provide?
Before the "flexibility" refactoring of the GMT, we had the TimeSeriesMassFlowTemperatures.mot and [time_series_mft_ets_coupling.py](https://github.com/urbanopt/geojson-modelica-translator/blob/old-design/geojson_modelica_translator/model_connectors/load_connectors/time_series_mft_ets_coupling.py). When refactoring these files were removed and not added back.

#### What does this PR accomplish?
Creates a new load, TimeSeriesMFT, and its couplings to heating and cooling ETSes. Also adds a test which uses a .mos file for the MFT data with stubbed network sources (ie infinite heating and cooling sources into ETSes). The graphical representation of this model at the district level is slightly broken. We have connections between the load's `buiMasTem` and TChi and THea set for the ETSes, but no `RealOutput`s for them.

#### What are the relevant tickets?
Resolves #287 
#### Screenshots (if appropriate)
Generated District System:
Screen Shot 2021-02-02 at 10 31 03 AM

Generated load:
Screen Shot 2021-02-02 at 10 19 34 AM

### CODE REVIEW

**Comment by macintoshpie** [comment]
@nllong the icon/ports have been updated to look more like the existing loads. But some things still look wonky:
> We have connections between the load's buiMasTem and TChi and THea set for the ETSes, but no RealOutputs for them.

I was thinking we'd hold off on adding real outputs for this, since you mentioned the proper location of massflowrate is undecided

**Comment by nllong** [comment]
I think we have some unnecessary port connections. I *think* it should look like this:
image

I'm still waiting for my dymola license--stay tuned.

File: geojson_modelica_translator/model_connectors/load_connectors/templates/getPeakMassFlowRate.mo [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+See
+<a href=\"modelica://Buildings.Experimental.DHC.Loads.BaseClasses.Validation.GetPeakMassFlowRate\">
+Buildings.Experimental.DHC.Loads.BaseClasses.Validation.GetPeakMassFlowRate</a>
+for how to invoke this function.
+</p>
+</html>",
+      revisions="<html>
+<ul>
+<li>
+August 20, 2020, by Hagar Elarga:<br/>
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/load_connectors/templates/getPeakMassFlowRate.mo:93]: thanks for including this!

File: geojson_modelica_translator/model_connectors/load_connectors/time_series_mft_ets_coupling.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
+"""
+****************************************************************************************************
+:copyright (c) 2019-2020 URBANopt, Alliance for Sustainable Energy, LLC, and other contributors.
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/load_connectors/time_series_mft_ets_coupling.py:3]: 2021! I'll update the copyrights...

File: tests/model_connectors/data/ts_mft_ets_example_data.mos [hunk 3]
```diff
@@ -2,7 +2,7 @@
 #Nominal heating water mass flow rate=67.82
 #Nominal chilled water mass flow rate=52.15
 #time,NODE 62:System Node Temperature[C],NODE 67:System Node Temperature[C],NODE 70:System Node Temperature[C],NODE 98:System Node Temperature[C],massFlowRateHeating,massFlowRateCooling
-double modelica(35041,7)
+double modelica(35040,7)
```
Comment (nllong) [inline tests/model_connectors/data/ts_mft_ets_example_data.mos:5]: interesting catch!

**Review by nllong — APPROVED** [review]
noice work everyone! I'm going to merge this one down and we can address issues with #303

@amyeallen1 -- I think this resolves #287

**Pull request merged**